en/docs/use-cases/streaming-usecase/extracting-data-from-static-sources-in-real-time.md

```diff
@@ -6,15 +6,15 @@ WSO2 Streaming Integrator can extract data from static sources such as databases
 
 A database table is a stored collection of records of a specific schema. Each record can be equivalent to an event. WSO2 Streaming Integrator can integrate databases into the streaming flow by extracting records in database tables as streaming events. This can be done via change data capture or by polling a database.
 
-![Extracting data from databases](../../assets/img/streaming/extracting-data-from-static-sources/extract-data-from-databases.png)
+![Extracting data from databases]({{base_path}}/assets/img/streaming/extracting-data-from-static-sources/extract-data-from-databases.png)
 
 To understand how data is extracted from a database into a streaming flow, consider a scenario where an online booking site automatically save all online bookings of vacation packages in a database. The company wants to monitor the bookings in real-time. Therefore, this data stored in the database needs to be extracted in real time. You can either capture this data as change data or poll the database. The `cdc` Siddhi extensions can be used for both methods as explained in the following subsections.
 
 ### Change data capture
 
 Change data capture involves extracting any change that takes place in a selected database (i.e., any insert, update or a deletion) in real-time.
 
-To capture change data via the [WSO2 Streaming Integrator Tooling](../../develop/streaming-apps/streaming-integrator-studio-overview.md), define an input [stream](https://siddhi.io/en/v5.1/docs/query-guide/#stream) with the appropriate schema to capture the information you require, and then connect a [source](https://siddhi.io/en/v5.1/docs/query-guide/#source) of the `cdc` type as shown in the example below.
+To capture change data via the [WSO2 Streaming Integrator Tooling]({{base_path}}/develop/streaming-apps/streaming-integrator-studio-overview.md), define an input [stream](https://siddhi.io/en/v5.1/docs/query-guide/#stream) with the appropriate schema to capture the information you require, and then connect a [source](https://siddhi.io/en/v5.1/docs/query-guide/#source) of the `cdc` type as shown in the example below.
```

Comment:
@RukshiW , from what I recall the links should not have ".md" at the end of the hyperlink. Please check this because these links may be broken